src/exercise/04.md

```diff
@@ -1,5 +1,11 @@
 # Creating custom components
```

Comment:
Guessing a fresh pull of `main` to bring your branch up to date will get rid of these changes you didn't make.